components/staticFeeds/sample-messages.json

```diff
@@ -118,6 +118,199 @@
       "confirmButton": {
         "label": "Got it!"
       }
+    },
+
+    {
+      "id": "demo-high-priority-no-group-no-date",
+      "title": "No group. No date. High priority.",
+      "titleShort": "No group. No date. High priority.",
```

Comment:
Might be nice to differentiate `title`, `description` and the `Short` versions of these, so that we can easily see where which is used in the UI.